src/components/map/MapboxMap.tsx

```diff
@@ -47,12 +57,14 @@ const MapboxGLMap = (): any => {
         //@ts-ignore
         container: mapContainerRef.current,
         style: baseMapStyle,
-        center: [0, 30],
-        zoom: 1,
+        center: [10, 30],
+        zoom: 2,
+        attributionControl: false,
+        antialias: true// enables MSAA antialiasing, to help make dotted line WHO borders more visible
```

Comment:
Are we sure this anti aliasing is good enough? Did the WHO advise you to do this?